assets/_sass/base/_typography.scss

```diff
@@ -114,6 +114,19 @@ textarea {
 	}
 }
 
+%labelsmall {
+  font-family: $font_inconsolata;
+  @include fontSize(16px);
+  line-height: 1.31;
+  letter-spacing: 1.3px;
+  text-transform: uppercase;
+
+  @include breakpoint('medium') {
+    @include fontSize(16px);
```

Comment:
You don't need to redeclare the font size on larger devices because it isn't changing between mobile & desktop. So you can remove line 125.

Reply:
removed line 125 here. it looks like atom-prettier removed some leading spaces, but only 125 has changed.